python/lsst/ip/diffim/psfMatch.py

```diff
@@ -38,6 +40,8 @@
 from . import diffimLib
 
 
+@deprecated(reason="This config class is no longer used. Will be removed after v27.",
+            version="v27.0", category=FutureWarning)
```

Comment:
Is this showing up in the docs and also during runtime?

Reply:
This will appear any time someone instantiates this class, including in the logs of pipelines that use it.

But this reminded me that I need to also deprecate the ConfigField that uses it in `PsfMatchConfig`.